src/main/java/com/hubspot/jinjava/el/TruthyTypeConverter.java

```diff
@@ -49,6 +61,12 @@ protected Double coerceToDouble(Object value) {
     if (value instanceof DummyObject) {
       return 0d;
     }
+
+    if (value instanceof String) {
+      String sanitizedValue = trimCommas((String) value);
+      return super.coerceToDouble(sanitizedValue);
+    }
```

Comment:
I'd prefer these to use similar logic as in `IntFilter` and `FloatFilter`. Make use of `NumberFormat#parse`